common/src/main/java/io/druid/math/expr/Expr.java

```diff
@@ -362,6 +362,13 @@ public ExprEval eval(ObjectBinding bindings)
   {
     ExprEval leftVal = left.eval(bindings);
     ExprEval rightVal = right.eval(bindings);
+
+    // Result of any Binary expressions is null if any of the argument is null.
+    // e.g "select null * 2 as c;" or "select null + 1 as c;" will return null as per Standard SQL spec.
+    if (!NullHandling.useDefaultValuesForNull() && (leftVal.isNull() || rightVal.isNull())) {
```

Comment:
I find it hard to mentally follow the phrase `useDefaultValuesForNull()`, plus negation in the beginning. What about making two methods with more vivid names instead,
 - `NullHandling.replace()`, the same as current `useDefaultValuesForNull()`,
 - `NullHandling.sqlCompatible()`, the same as current `!useDefaultValuesForNull()`?

Reply:
done.